templates/themes/app/pagelayout/header.html.twig

```diff
@@ -2,9 +2,9 @@
     <div class="container">
         {% include '@ibexadesign/content/parts/site_logo.html.twig' %}
 
-        <a class="mainnav-toggle" href="#">
+        <button class="mainnav-toggle">
```

Comment:
ovdje bi bilo dobro dodati 

- aria-expanded="false" (promijeniti na klik na true)
- aria-controls="main-navigation" -> samo što se to referencira na id, a ne klasu pa treba i na taj element dodati id
- aria-label="Open main navigation"

Reply:
fixed in https://github.com/netgen/media-site/pull/214/commits/f1fbfc5dc05bb50aef14f755dd47c099f632e6b7